Added the README to the project

francislaus opened the pull request:
All important parts of a README were added, which includes the following points
* Description of the project
* Running the project
* Statement of contributions

This should resolve #12

francislaus committed:
Add feature to README for #12

Added the 'Statement of Contributions' to the README

emilelm96 committed:
Added a general and a more thorough description. #12

Also corrected some small typos. Writing the even more thorough description still remains.

apeinot committed:
Add information about compiling and running in README

Complete the README with explanation on how to compile and run the project.
This is part of the solution for #12

francislaus committed:
Fixed README for #12

Fixed some small differences in how to write captions so that they
comply with each other

emilelm96 committed:
Added a description of the e-mail notifications #12

Like specified in the grading specification we need to describe how the notifications were implemented and unit tested. I did this for my part, the e-mail notifications.

francislaus commented:
The changes of @emilelmarsson look good

francislaus committed:
Fixed github notifications for #12

Added the documentation for the github notifications in the README as it
is required in the task assignment

Samuelhertzberg commented:
Looks very good. However: "the students document the build list URL in the README."

emilelm96 committed:
Added contributions to Emil and Franz. #12

Emil and Franz implemented the web interface.

apeinot committed:
Develop some part of the README and complete the statement of contributions

This is part of solution for #12

Samuelhertzberg committed:
Added web interface info to readme

Added another header with info regarding the history and web interface. 
It now specifies the URL to access the hustory.
Also added jonathan as a web interface collaborator.

Samuelhertzberg committed:
Merge branch 'issue12' of github.com:apeinot/DD2480_lab2_group18 into issue12
